pkg/lib/cockpit.d.ts

```diff
@@ -131,8 +131,17 @@ declare module 'cockpit' {
         close(): void;
     }
 
+    type VariantType = string | string[] | Uint8Array | number | number[] | boolean | boolean[];
+    interface Variant {
+        v: VariantType
```

Comment:
missing `;`

Reply:
to my eye I'd also reverse the order of the `t` and `v` paramters here to match the `variant()` function call, but it's a matter of taste (and even the body of that call does it the other way)...

Reply:
Makes sense